domain/infosync/info.go

```diff
@@ -67,11 +67,12 @@ type InfoSyncer struct {
 // It will not be updated when tidb-server running. So please only put static information in ServerInfo struct.
 type ServerInfo struct {
 	ServerVersionInfo
-	ID         string `json:"ddl_id"`
-	IP         string `json:"ip"`
-	Port       uint   `json:"listening_port"`
-	StatusPort uint   `json:"status_port"`
-	Lease      string `json:"lease"`
+	ID            string `json:"ddl_id"`
+	IP            string `json:"ip"`
+	Port          uint   `json:"listening_port"`
+	StatusPort    uint   `json:"status_port"`
+	Lease         string `json:"lease"`
+	BinlogEnabled bool   `json:"binlog_enabled"`
```

Comment:
If the `tidb-server` is disconneted with `Pump` and the user chooses to ignore the error, will the `tidb-server` update the `binlog_enabled` to false?

Reply:
fixed PTAL again